src/main/java/emis/Storage.java

```diff
@@ -0,0 +1,91 @@
+package main.java.emis;
+import main.java.emis.exceptions.EmisException;
+import main.java.emis.command.*;
+import main.java.emis.task.*;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.io.File;
+import java.io.FileWriter;
+
+// Storage class, deals with loading tasks and saving tasks
+public class Storage {
+    private String filePath;
+    private ArrayList<Task> al;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+        this.al = new ArrayList<>();
+    }
+
+    public ArrayList<Task> load() throws EmisException {
+        try {
+            Scanner sT = new Scanner(new File(this.filePath));
+            while (sT.hasNextLine()) {
+                // read from file, add to arraylist al
+                String input = sT.nextLine();
+                String[] data = input.split("\\|");
```

Comment:
Should the name for String[] be in singular form or plural? Same for the ArrayList<Task> al above